KarhooUISDK/Screens/CheckoutScreen/Checkout/KarhooCheckoutPresenter.swift

```diff
@@ -31,6 +31,8 @@ final class KarhooCheckoutPresenter: CheckoutPresenter {
     private var bookingRequestInProgress: Bool = false
     private var flightDetailsScreenIsPresented: Bool = false
     private let baseFareDialogBuilder: PopupDialogScreenBuilder
+    private var passengerCount: Int = 1
+    private var luggageCount: Int = 0
```

Comment:
`passengerCount` and `luggageCount` should not have default values, since there is no runtime where they are actually used.